packages/smarthr-ui/src/components/Textarea/Textarea.tsx

```diff
@@ -98,6 +99,20 @@ const textarea = tv({
   },
 })
 
+const calculateIdealRows = (
+  element?: HTMLTextAreaElement | null,
+  maxRows: number = Infinity,
```

Comment:
nits）ほぼ意味はないのですが、安全に扱える整数値に上限があるため、以下のほうが厳密には良いかもしれません（？
```suggestion
  maxRows: number = Number.MAX_SAFE_INTEGER,
```

Reply:
おおおお！！そうなのですね…！勉強になります！